manifests/0000_07_cluster-network-operator_01_crd.yaml

```diff
@@ -67,14 +67,27 @@ spec:
               type: array
               items:
                 type: object
-                required: ["type", "name", "rawCNIConfig"]
+                required: ["type", "name"]
                 properties:
                   type:
                     type: string
                   name:
                     type: string
                   rawCNIConfig:
                     type: string
+                  macVlanConfig:
+                    type: object
+                    required: ["master", "ipam"]
```

Comment:
I would say that ipam should not be required - if omitted, it defaults to "dhcp"

Reply:
Sure. Changed.